5.6.36-apache/apache2.conf

```diff
@@ -15,6 +15,9 @@ LogLevel warn
 IncludeOptional mods-enabled/*.load
 IncludeOptional mods-enabled/*.conf
 
+IncludeOptional conf-enabled/*.conf
```

Comment:
Is this ok? We need to review what conf and what sites we are enabling..

Reply:
yes, the lines were just moved up.  I haven't figured out why this fixes it yet.